src/test/java/io/github/externschool/planner/repository/RoleRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package io.github.externschool.planner.repository;
+
+import io.github.externschool.planner.entity.Role;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+public class RoleRepositoryIntegrationTest {
+    @Autowired
+    private RoleRepository roleRepository;
+
+    @Test
+    public void testTableContainsPresetRoles() {
+        List<String> names = new ArrayList<>(Arrays.asList("admin", "guest", "officer", "student", "teacher"));
+        List<Role> roles = readRoles();
+
+        assertThat(roles).isNotEmpty();
+        assertThat(roles.size() == 5);
+        for (String name: names) {
+            assertThat(roles.contains(new Role(name)));
+        }
+    }
+
+    @Test
+    public void testCreateReadUpdateDeleteRoles() {
```

Comment:
This test is too complicated to understand.

One test - one main check. Instead you checked three things in one test